sdk/python/kfp/dsl/_container_op.py

```diff
@@ -745,6 +746,17 @@ def add_volume(self, volume):
         self.volumes.append(volume)
         return self
 
+    def add_toleration(self, tolerations):
```

Comment:
Could you add a basic unit test to cover the method: calling the method and checking the output yaml contains the correct tolerations config? This will help us to make sure future changes won't break existing feature.

Reply:
sure will do.

Reply:
You'll probably need to test the `op_to_template` method for this.